demo/addons/fmod/tool/property_editors/FmodEventEditorProperty.gd

```diff
@@ -0,0 +1,84 @@
+@tool class_name FmodEventEditorProperty extends FmodPathEditorProperty
+
+
+static var EVENT_PARAMETER_PREFIX_FOR_PROPERTIES = "event_parameter"
+static var fmod_parameter_ui: PackedScene = load("res://addons/fmod/tool/property_editors/FmodParameterUi.tscn")
+
+var former_event_description: FmodEventDescription
+
+func initialize(open_project_explorer_callable: Callable, path_prop: String, guid_prop: String):
+	super(open_project_explorer_callable, path_prop, guid_prop)
+
+func _update_property():
+	super()
+	_update_parameters()
+	var event_description: FmodEventDescription = FmodServer.get_event_from_guid(get_edited_object().event_guid)
+	former_event_description = event_description
+
+func _set_path_and_guid(path: String, guid: String):
+	super(path, guid)
+	_update_parameters()
+	former_event_description = FmodServer.get_event_from_guid(get_edited_object().event_guid)
+
+func _update_parameters():
+	print("update params")
```

Comment:
Is this for debugging? Using `print_debug` might be more useful, as it gets omitted in Production builds.